right-click-for-superpowers/script.js

```diff
@@ -0,0 +1,117 @@
+import {FilesetResolver, LlmInference} from 'https://cdn.jsdelivr.net/npm/@mediapipe/tasks-genai';
+
+const MODEL_FILE_NAME = 'https://storage.googleapis.com/jmstore/kaggleweb/grader/g-2b-it-gpu-int4.bin';
+
+let promptText = '';
+let llmInference = undefined;
```

Comment:
let llmInference; will be enough

Reply:
Updated though I like to be explicit so anyone reading knows its state incase they dont know the default value is that.